obs/project.go

```diff
@@ -91,3 +97,102 @@ type RepositoryPath struct {
 	Project    string `json:"project" xml:"project,attr"`
 	Repository string `json:"repository" xml:"repository,attr"`
 }
+
+func (c Client) CreateProject(project *Project) error {
+	xmlData, err := xml.MarshalIndent(project, "", " ")
+	if err != nil {
+		return err
```

Comment:
Can we extend this error with some additional information, e.g.

```suggestion
		return fmt.Errorf("creating obs project: marshalling project meta: %w", err)
```